jaxopt/_src/isotonic.py

```diff
@@ -89,7 +89,18 @@ def _isotonic_l2_pav(y):
 
 
 @jax.custom_jvp
-def isotonic_l2_pav(y):
+def _isotonic_l2_pav(y):
+  if not NUMBA_AVAILABLE:
+    warnings.warn(
+        "Numba could not be imported. Code will run much more slowly."
+        " To install, run 'pip install numba'."
+    )
+  # Define the expected shape & dtype of output.
+  shape_dtype = jax.ShapeDtypeStruct(shape=y.shape, dtype=y.dtype)
+  sol = jax.pure_callback(_isotonic_l2_pav_numba, shape_dtype, y)
+  return sol
+
+def isotonic_l2_pav(y, y_min=None, y_max=None, increasing=True):
   """Solves an isotonic regression problem using PAV.
 
```

Comment:
Document y_min, y_max and increasing (you need to write the equations in the two cases)